Source/Atomic/Atomic2D/ParticleEmitter2D.cpp

```diff
@@ -312,8 +316,7 @@ void ParticleEmitter2D::Update(float timeStep)
             emitParticleTime_ -= timeBetweenParticles;
         }
 
-        if (emissionTime_ > 0.0f)
-            emissionTime_ = Max(0.0f, emissionTime_ - timeStep);
+        emissionTime_ = Max(0.0f, emissionTime_ - timeStep);

```

Comment:
Well, that line makes no practical difference whatsoever, and you never know if compilers will be smart enough to remove that redundancy, probably not since they'd have to crawl throw other functions to check if they change that var between the two checks. @JoshEngebretson can I guard exclusions? If by 'upstream' you're talking about Urho, it's no redundant in the code there because we're the ones who modified the line 303 which rendered that check I removed redundant. It was changed from >= to > so I wonder why it's like that in the Urho code, this line: https://github.com/urho3d/Urho3D/blob/master/Source/Urho3D/Urho2D/ParticleEmitter2D.cpp#L315 will only step in when emissionTime_ is exactly zero what's kinda weird because counting on floats comparisons to exact values isn't exactly good practice but in this case since the value probably will never be a result of complex arithmetic I guess it's acceptable.
@JimMarlowe even if that is lost upon a rebase/merge at least we'll see the benefits of less redundancy in the mean time 🐫🐫.